### PR TITLE
End of Year: add prompt

### PR DESCRIPTION
**Pull request by leandroalonso**
| 📘 Project: #376 |
|:---:|

This PR adds a modal that prompts the user to check their End of Year. Design and iPad are not covered in this task and will be done in a future PR.

## To test

1. On `FeatureFlag.swift` change `endOfYear` to `true`
2. Run the app
3. ✅ Check that a bottom sheet appears with two buttons: "View My 2022" and "Not Now"
4. ✅ Tap "Not Now" and ensure the bottom sheet is dismissed
5. Re-run the app
6. When the bottom sheet appears, tap "View My 2022"
7. ✅ Ensure a VC is shown in `fullScreen` mode

## Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [x] I have considered adding unit tests for my changes.
- [x] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

File: PocketCastsTests/UnitTests.xctestplan [hunk 1]
```diff
@@ -13,6 +13,7 @@
     }
   ],
   "defaultOptions" : {
+    "codeCoverage" : false,
```
Comment (leandroalonso) [inline PocketCastsTests/UnitTests.xctestplan:16]: I had to add this to make SwiftUI previews work. 🤷 

More details on the commit: https://github.com/Automattic/pocket-casts-ios/pull/377/commits/3142e082d89b477bbecfbdaabab60cc11f47c56d

File: podcasts/FeatureFlag.swift [hunk 2]
```diff
@@ -12,4 +12,6 @@ enum FeatureFlag {
 
     /// Whether logging of Firebase events in console are enabled
     static let firebaseLoggingEnabled = false
+
+    static let endOfYear = false
```
Comment (emilylaguna) [inline podcasts/FeatureFlag.swift:17]: Nit: Can we add a comment for this?

File: podcasts/End of Year/Views/EndOfYearModal.swift [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                HStack {
+                    Spacer()
+                    Text("View My 2022")
+                    Spacer()
+                }
+            }
+            .textStyle(RoundedButton())
+            .contentShape(Rectangle())
+
+
```
Comment (emilylaguna) [inline podcasts/End of Year/Views/EndOfYearModal.swift]: Nit: Can we remove this double space? 

```suggestion

```

**Review by emilylaguna — APPROVED** [review]
👍 Nice work, I left some small nitpick comments. But they're not blockers since we're still working on this feature.

**Pull request merged**